tests/Unit/Booking/BookingUnitTest.php

```diff
@@ -29,74 +29,14 @@ public function itReturnsThePositionRelation()
     }
 
     /** @test */
-    public function itCorrectlyDeterminesIfABookingCanBeMade()
+    public function itReturnsTheUserRelation()
     {
-        $this->booking->from = new Carbon('10th January 2018 15:00:00');
-        $this->booking->to = new Carbon('10th January 2018 16:00:00');
-        $this->booking->save();
+        $this->booking->setRelation('user', User::initModelWithData([
```

Comment:
I don't think brute setting the relation is the best approach here. I would just mock the user as we have done before with the data, and assert that is what we are getting through the relation.

Reply:
I disagree - this is effectively mocking it in a more explicit way.

Reply:
hmm. I think this would be better in tests that don't have the sole purpose of checking that the the relationship works. I personally think that mocking the outcome of the user::find method is more authentic in checking that the relationship is setup properly. To me this is like asserting that true == true

Reply:
as discussed, fixed